www/nextcloud-client/Portfile

```diff
@@ -6,9 +6,11 @@ PortGroup               qt5 1.0
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               github 1.0
 
-github.setup            nextcloud desktop 2.5.1 v
 fetch.type              git
 name                    nextcloud-client
+version                 2.6.4
+revision                0
+github.setup            nextcloud desktop ${version} v
```

Comment:
the GitHub PG sets the version itself, so you change here to explicitly set it first is unnecessary. How it was before was correct so please revert back to that way.

Reply:
Thanks for the reply. I only looked at the [documentation](https://trac.macports.org/wiki/howto/Upgrade) and thought, that this would be the way to go. Next time I know better.